src/app/about/page.js

```diff
@@ -3,34 +3,95 @@ import Logo from "../../../public/assets/logo.svg";
 import Image from "next/image";
 import { sections } from "@/data/about/Sections.js";
 import Sun from "@/public/assets/sun.svg";
+import { motion } from "framer-motion";
+
+const animations = {
+  start: {
+    opacity: 0,
+    y: 30,
+  },
+  end: {
+    opacity: 1,
+    y: 0,
+  },
+};
+
+const sunRight = {
+  start: {
+    opacity: 0,
+  },
+  end: {
+    opacity: 1,
+  },
+};
+
+const sunLeft = {
+  start: {
+    opacity: 0,
+  },
+  end: {
+    opacity: 1,
+  },
+};
 
 const About = () => {
   return (
     <div className="pb-56 overflow-hidden">
-      <Image
-        src={Logo}
-        alt="ISA Icon"
-        className="py-8 mx-auto h-[400px] w-[400px]"
-      />
-      <div className="w-full relative flex flex-row justify-center items-center">
+      <motion.div
+        variants={animations}
+        initial="start"
+        whileInView="end"
+        transition={{ delay: 0.2 }}
+      >
         <Image
-          src={Sun}
-          alt="Blue sun"
-          className="hidden xl:block absolute -right-[25%] scale-140"
-        />
-        <Image
-          src={Sun}
-          alt="Blue sun"
-          className="hidden xl:block absolute top-[500px] -left-[25%] scale-140"
+          src={Logo}
+          alt="ISA Icon"
+          className="py-8 mx-auto h-[400px] w-[400px]"
         />
+      </motion.div>
+
+      <div className="w-full relative flex flex-row justify-center items-center">
+        <motion.div
+          variants={sunRight}
```

Comment:
```suggestion
          variants={sun}
```


you can reuse the same animation objects for each motion (see comment above)